src/components/Message/MessageButton.js

```diff
@@ -41,16 +41,26 @@ const StyledMessageButton = styled.a`
 `;
 
 class MessageButton extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this._handleOnClick = () => {
+      const { label, onClick } = this.props;
+      onClick(label);
+    };
+  }
+
   static propTypes = {
     /** Button's label */
     label: PropTypes.string,
```

Comment:
Lets add a `value` too so that we can store some state on the button besides its label and have it sent to the onClick

Reply:
@ekampf Fixed with 4dedee1